composer.json

```diff
@@ -16,11 +16,16 @@
         "composer/installers": "*",
         "openbuildings/phpunit-spiderling": "^0.3",
         "openbuildings/db-fixtures": "^0.2 || ^0.3",
-        "kohana/core": "^3.3",
-        "phpunit/phpunit": "^7"
```

Comment:
@stanstst @tumbalev This is not accurate. PHPUnit is not a dev dependency for this testing library.

Reply:
@hkdobrev I think that we need the PHPUnit to test the functionality of this package only, so for me it makes sense to be require-dev. Isn't it?

Reply:
This package is a testing library that depends on PHPUnit classes when used. So if PHPUnit is a dev dependency, that would allow to install it with a different PHPUnit version in your project, because Composer doesn't take into account dev dependencies of dependencies of course.

And it just won't work as the classes this library depends on are from a specific PHPUnit version and might break more often between major versions.

Reply:
Could you please show me the exact place of dependency because I can't spot it. There are no extends, neither constructors, so the only place could be `spl_autoload_register(array('Functest_Tests', 'autoload'));`, but I'm not sure as the IDE drives me to the class `classes/Functest/Tests`. Also, there is no such class in the main library, so if this is the dependency it means that there is no test which will catch that.

Reply:
@tumbalev These are all places that can be broken and have actually most needed changes in the past due to changes between PHPUnit versions:

https://github.com/OpenBuildings/functest/blob/88ee629a6f6f94b666c1fbb742adadac7e8333dd/classes/Kohana/Functest/Tests.php#L134
https://github.com/OpenBuildings/functest/blob/88ee629a6f6f94b666c1fbb742adadac7e8333dd/classes/Kohana/Testcase/Functest.php#L18
https://github.com/OpenBuildings/functest/blob/88ee629a6f6f94b666c1fbb742adadac7e8333dd/classes/Kohana/Testcase/Functest.php#L20
https://github.com/OpenBuildings/functest/blob/88ee629a6f6f94b666c1fbb742adadac7e8333dd/classes/Kohana/Testcase/Functest.php#L35

Also, see https://github.com/OpenBuildings/functest/pull/6

Reply:
Ok, thanks.